dali/python/backend_impl.cc

```diff
@@ -103,7 +103,14 @@ void ExposeTensor(py::module &m) { // NOLINT
       R"code(
       Tensor residing in the CPU memory.
       )code")
-    .def("shape", &Tensor<CPUBackend>::shape,
+    .def("shape",
+        [](Tensor<CPUBackend> &t) -> py::list {
```

Comment:
Please make it a function - or two functions:
```
template <int ndim>
py::list as_py_list(const TensorShape<ndim> &shape) {
          py::list ret(shape.size());
          for (int i = 0; i < shape.size(); i++) {
            ret[i] = shape[i];
          }
          return ret;
}

template <typename Backend>
auto py_shape(const Tensor<Backend> &tensor) { return as_py_lisy(tensor.shape); }
```

Reply:
Done